src/core/backbone_router/leader.hpp

```diff
@@ -126,11 +136,37 @@ class Leader : public InstanceLocator
      */
     bool HasPrimary(void) const { return mConfig.mServer16 != Mac::kShortAddrInvalid; }
 
+    /**
+     * This method gets the Domain Prefix in the Thread Network.
+     *
+     * @retval A pointer to the Domain Prefix or NULL if there is no Domain Prefix.
+     *
+     */
+    const otIp6Prefix *GetDomainPrefix(void) const { return (mDomainPrefix.mLength == 0) ? NULL : &mDomainPrefix; }
```

Comment:
nit: return alias and leave it for the callers to check if the domain prefix is valid?
```suggestion
    const otIp6Prefix &GetDomainPrefix(void) const { return mDomainPrefix; }
```